src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block">
+      <ul class="stars--0">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+      <ul class="stars--1">
```

Comment:
Missing the _Block_ name without the _Modifier_. This class should be:
`<ul class="stars stars--1">`

This is valid for all blocks you created